students/travis_nelson/lesson07/html_render.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+
+# This is the framework for the base class
+class Element(object):
+
+    tag = "html"
+    indent = ""
+
+    def __init__(self, content=None, **element_attribute):
+
+        if content is not None:
+            self.contents = [content]
+        else:
+            self.contents = [""]
+        if element_attribute:
+            self.attributes = element_attribute
+
+    def append(self, new_content):
+        self.contents.append(new_content)
+
+    def _open_tag(self):
+        try:
+            open_tag = [f"<{self.tag}"]
+            for k, v in self.attributes.items():
+                open_tag.append(f' {k}="{v}"')
+            open_tag.append(">")
+            return "".join(open_tag)
+        except AttributeError:
+            return f"<{self.tag}>"
```

Comment:
per my comment above, if `self.attributes` will be `{}` then you can remove except clause and your code should work as is